features/bind.feature

```diff
@@ -1,4 +1,6 @@
 Feature: Binding
+  
+  @conjur-version-5
```

Comment:
this doesn't seem great - we're effectively only testing bind now on v5, though we are still aiming to support v4. let's log an issue to address testing binding for v4 and v5 before we cut the next release

Reply:
Thanks for calling that out, I made the change here but had forgotten to make a note to circle back to that. The issue for correcting this is here: https://github.com/cyberark/conjur-service-broker/issues/79